components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/util/APIMgtConstants.java

```diff
@@ -374,7 +374,7 @@
         public static final String API_CREATE = "API_CREATE";
         public static final String API_UPDATE = "API_UPDATE";
         public static final String API_DELETE = "API_DELETE";
-        public static final String API_PUBLISH = "API_PUBLISH";
+        public static final String API_STATE_CHANGE = "API_STATE_CHANGE";
```

Comment:
This is a public variable and changing this will break the backward compatibility (Major version change). Are we exposing this class (export packages). If so, then we are breaking the version strategy,